src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +49,18 @@ public List<IexLastTradedPrice> getLastTradedPrice(
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  /**
+   * Get historical pricing data for the given symbol on the given date
+   *
+   * @param symbol the symbol to retrieve data about
+   * @param date the date which should be queried
+   * @return historical pricing data for the given symbol on the given date
+   */
+  @GetMapping(value = "${mvc.iex.getHistoricalPricePath}", produces = {
+          MediaType.APPLICATION_JSON_VALUE})
+  public List<IexHistoricalPrice> getHistoricalPrice(
+          @RequestParam(value = "symbol") final List<String> symbol,
+          @RequestParam(value = "date") final List<String> date) {
```

Comment:
Right now for `date` we just pass the value through to the IEX API, but sometimes we end up needing to do some date-based logic, which is easier when using an actual datetime class rather than a String. Try using one of the Java 8 datetime options here as well.